spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/secretManager/GoogleSecretComparatorByVersion.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.config.server.environment.secretManager;
```

Comment:
Spring doesn't do camel case in package names. `secretmanager` is fine.

Reply:
Ok, done.